packages/mdx/package.json

```diff
@@ -34,7 +34,7 @@
     "mdxast"
   ],
   "dependencies": {
-    "@babel/core": "7.8.4",
+    "@babel/core": "7.9.0",
```

Comment:
It looks like these versions were pinned here https://github.com/mdx-js/mdx/pull/644

But I'm wondering if there's any reason to not use `^` ranges on these dependencies?